core/src/i18n/en.yaml

```diff
@@ -127,7 +126,26 @@ services:
 settings:
   confirmations: Confirmations
   dontConfirmDelete: Delete without confirmation
+common:
+  name: Name
+  created: Created
+  labels: Labels
+  status: Status
   language: Language
+  buttons:
+    delete: Delete
+    cancel: Cancel
+    edit-yaml: Edit YAML
+  delete-dialog:
+    title: Remove {{name}}
+    message: Are you sure you want to delete {{type}} {{name}}?
+    delete-confirm: Don't show delete confirmations
+  tooltips:
+    copy-button: Copy to clipboard
+    duplicate-key: Duplicate key
+    k8s-name-input: Name must contain lower case alphanumeric characters, can contain '-' (like 'my-name1').
```

Comment:
Restated [here](https://github.com/kyma-project/busola/pull/323#discussion_r685961290)

Reply:
That was my suggestion, in order to reduce characters in tooltips. 
One alphanumeric character seems to be enough as valid entry, so "must contain at least one lowercase alphanumeric character" is correct. But I'm fine with your phrasing too, it's not extremely long. We just should keep them all consistent.

Reply:
What I actually mean here is these are two different meanings:

- `must contain` means `must have at least one, but if it does, it can also have uppercase letters`
- `must consist of` means that all the characters must be lowercase letters and numbers (+ a hyphen, as stated further) and uppercase is not allowed

I suspect that the intended meaning was the latter, hence my comment.